packages/palette_generator/example/lib/main.dart

```diff
@@ -237,32 +242,32 @@ class PaletteSwatch extends StatelessWidget {
   /// If the [color] argument is omitted, then the swatch will show a
   /// placeholder instead, to indicate that there is no color.
   const PaletteSwatch({
-    Key key,
-    this.color,
+    Key? key,
+    required this.paletteColor,
     this.label,
   }) : super(key: key);
 
-  /// The color of the swatch. May be null.
-  final Color color;
+  /// The color of the swatch.
+  final PaletteColor paletteColor;
```

Comment:
Why did this need to change type and name? Seems like an unnecessary breaking change.

Reply:
it was an attempt to make `selectedSwatches` and `Color`s non-nullable (https://github.com/flutter/packages/pull/287#discussion_r577079454)

ye I'm stuck on this and not sure how to solve it without breaking changes

Reply:
reverted back